### PR TITLE
Intermine Enhancements

### PR DESCRIPTION
**Pull request by blankenberg**

### CODE REVIEW

**Comment by blankenberg** [comment]
@yochannah

**Review by jmchilton — APPROVED** [review]

File: tools/data_source/intermine.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<!--
+    If the value of 'URL_method' is 'get', the request will consist of the value of 'URL' coming back in
+    the initial response.  If value of 'URL_method' is 'post', any additional params coming back in the
+    initial response ( in addition to 'URL' ) will be encoded and appended to URL and a post will be performed.
+-->
+<tool name="InterMine" id="intermine" tool_type="data_source" version="1.0.0">
+    <description>server</description>
+    <command interpreter="python">data_source.py $output $__app__.config.output_size_limit</command>
```
Comment (bgruening) [inline tools/data_source/intermine.xml:9]: should this be quoted?

File: config/tool_data_table_conf.xml.sample [hunk 2]
```diff
@@ -100,4 +100,9 @@
         <columns>value, name, url</columns>
         <file path="tool-data/biom_simple_display.loc" />
     </table>
+    <!-- simple intermine servers -->
+    <table name="intermine_simple_display" comment_char="#" allow_duplicate_entries="False">
+        <columns>value, name, url</columns>
+        <file path="tool-data/intermine_simple_display.loc" />
```
Comment (jmchilton) [inline config/tool_data_table_conf.xml.sample:106]: Is there a sample loc for this that should be added?

Comment (blankenberg) [reply config/tool_data_table_conf.xml.sample:106]: Ooops. Yes, there is. I'll add it from my laptop after I get home today.

**Pull request merged**